schematic/models/validate_attribute.py

```diff
@@ -524,6 +661,33 @@ class ValidateAttribute(object):
     def __init__(self, dmge: DataModelGraphExplorer) -> None:
         self.dmge = dmge
 
+    def get_no_entry(self, entry, node_display_name):
+        # check if <NA> in list let pass if not required and no value is recorded.
+        no_entry = False
+        # if not col_is_required:
+        na_allowed = GenerateError._get_is_na_allowed(
+            node_display_name=node_display_name, dmge=self.dmge
+        )
+        value_is_na = GenerateError._get_error_value_is_na(
```

Comment:
If you are usign these methods from a different class, they shouldn't be priavte